tools/cli/internal/openapi/filter/code_sample.go

```diff
@@ -121,6 +125,41 @@ func newAtlasCliCodeSamplesForOperation(op *openapi3.Operation) codeSample {
 	}
 }
 
+func (f *CodeSampleFilter) newGoSdkCodeSamplesForOperation(op *openapi3.Operation, opMethod string) codeSample {
+	version := strings.ReplaceAll(apiVersion(f.metadata.targetVersion), "-", "") + "001"
+	operationID := cases.Title(language.English, cases.NoLower).String(op.OperationID)
+	sdkCall := fmt.Sprintf(
+		"sdk.%sApi\n    .%sWithParams(ctx, params)\n    .Execute()",
+		strings.ReplaceAll(op.Tags[0], " ", ""), operationID)
+
+	switch opMethod {
+	case "GET", "POST", "PATCH", "PUT":
+		sdkCall = "  sdkResp, httpResp, err := " + sdkCall
+	case "DELETE":
+		sdkCall = "  httpResp, err := " + sdkCall
+	}
+
+	source := "import (\n" +
```

Comment:
Suggestion, no strong opinion: use go templates to make this more readable.

Reply:
good point 👍

Reply:
I updated the logic to use go template. I still don't use the template in the unit test to check that the template has the valid format.

Reply:
updated

Reply:
Thanks for the changes!